integration-tests/chopsticks/src/network/spiritnet.ts

```diff
@@ -26,6 +26,14 @@ export function setGovernance(addr: string[]) {
 	}
 }
 
+export function setSafeXcmVersion3() {
```

Comment:
This only returns something, right? It does not `set` anything, as far as I can see.

Reply:
Yes, not the best naming. I renamed in: [3c3d182](https://github.com/KILTprotocol/kilt-node/pull/668/commits/3c3d18262b2ce6f587e39bb1d2a35a8e8fade35f)

Reply:
Let's go with `getSafeXcmVersion3StorageEntry`, and make it unambiguous.

Reply:
Or `XcmPalletSafeVersion3StorageEntry`.

Reply:
[2140bc7](https://github.com/KILTprotocol/kilt-node/pull/668/commits/2140bc757ec924620ee2018b0b1d98275d078015)

Reply:
I guess TS practice is to start the name lowercase 😇